Tests/Fennec_Enterprise_XCUITests.xctestplan

```diff
@@ -59,8 +59,6 @@
         "DragAndDropTestIpad\/testTryDragAndDropHistoryToURLBar()",
         "FindInPageTests\/testFindFromMenu()",
         "FirstRunTourTests\/testFirstRunTour()",
-        "HistoryTests\/testClearPrivateDataButtonDisabled()",
```

Comment:
I really guessed that removing these lines from the `.xctestplan`'s `skippedTests` section could enable all HistoryTests.

Reply:
There are tests disabled in this test plan, [Fennec_Enterprise_XCUITests.xctestplan](https://github.com/mozilla-mobile/firefox-ios/pull/12181/files#diff-a229eaaced1580e59ded7824941209df3eeb8b39b893710ff1675cdca6415f2e) which is the Full_Functional_Test test plan because they are running in the Smoketest test plan. If we enable them here, we are running them twice and there is not need. So, you can undo this change